src/frontend/bibletime_slots.cpp

```diff
@@ -63,6 +63,12 @@ void BibleTime::slotSettingsOptions() {
                    refreshDisplayWindows();
                    refreshProfileMenus();
                    m_infoDisplay->updateColors();
+
+#ifdef BUILD_TEXT_TO_SPEECH
```

Comment:
Please indent the preprocessor directives here and in other code blocks as well.

Reply:
Is the indentation correct in the new version?

Reply:
Yes, thanks! 😃